specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2017-04-01-preview/examples/Subscription_GetOperationResults.json

```diff
@@ -0,0 +1,27 @@
+{
+    "parameters": {
+      "api-version": "2017-04-01-preview",
+      "subscriptionId": "56b5e0a9-b645-407d-99b0-c64f86013e3d",
+      "location": "West US",
+      "operationId": "637508167933711791.bf67d4f3-8603-4a4e-bcd5-be9f2e32caab"
+    },
+    "responses": {
+      "200": {
+        "body": {
+            "status": "Success",
+            "outputUri": "http://myoutput.com"
+        }
+      },
+      "202": {
+        "body": {
+            "status": "Started"
+        }
+      },
```

Comment:
I guess this is a result returned from the URI in Azure-AsyncOperation header, right? Then please follow the schema in rpc. [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#operation-resource-format](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#operation-resource-format)
(i.e. outputUri should be under properties)